code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -253,12 +252,16 @@ GLOBAL_LIST_INIT(strain_list, init_glob_strain_list())
 	return strain_list
 
 ///returns a list of strains(xeno castedatum paths) that this caste can currently evolve to
-/datum/xeno_caste/proc/get_strain_options()
-	var/datum/xeno_caste/root_type = type
+/proc/get_strain_options(datum/xeno_caste/root_type)
+	RETURN_TYPE(/list)
+
+	ASSERT(ispath(root_type), "Bad root type passed to get_strain_options")
 	while(initial(root_type.parent_type) != /datum/xeno_caste)
 		root_type = root_type::parent_type
 	var/list/options = GLOB.strain_list[root_type]
-	return options?.Copy()
+	options = options.Copy()
+	options += root_type
+	return options
```

Comment:
We don't actually need the options var at all.
list + thing just creates a new list. I can't suggest edit to remove the option line above though.
```suggestion
	return GLOB.strain_list[root_type] + root_type
```